### ISSUE TITLE
Notes field for Caver & Cave

### ISSUE THREAD

**Issue by IdlePhysicist** [IdlePhysicist/cave-logger#27]
Think about adding a note field to the caver/person table. The same could be done for caves/locations too. In fact that might be of more use.

These notes should only be viewable through an inspection view. Which would mean re-implementing that for the Caver and Cave panes.

---

Luckily I had just commented out most of the GUI code for displaying inspection windows. The `db` and `model` package will require more work along with the Python scripts which generate the databases.

I should point out that this will be a breaking change, meaning that people’s existing databases will not work and will need to be migrated. A migration could potentially be done in Python, meaning the script would read the file and determine if it had the right columns, then insert those columns if they are missing. I will need to brush up on SQLite to see how to do that.

**Comment by IdlePhysicist** [comment]
```
ALTER TABLE table_name ADD new_column_name column_definition;
```

**Comment by IdlePhysicist** [comment]
~~Since this would be a breaking change it might be best to schedule this for a v2.1.0 milestone.~~
It would only break if the app tried to read an old spec database.

**Comment by IdlePhysicist** [comment]
Closed by PR #28
